C/guide/ch_accts.xml

```diff
@@ -369,19 +373,23 @@ Translators:
 
       <para>Below are the standard <guilabel>Income</guilabel> and <guilabel>Expense</guilabel> accounts after
         selecting <guilabel>Common Accounts</guilabel> in the assistant for creating a new Account
-        Hierarchy (<menuchoice>
+        Hierarchy (
+        <menuchoice>
           <guimenu>Actions</guimenu><guimenuitem>New Account Hierarchy...</guimenuitem>
-        </menuchoice>).
+        </menuchoice>
+        ).
```

Comment:
Das dürfte unerwünschte Leerzeichen in der Klammer ergeben, aber macht die Klammer überhaupt Sinn?  Gab es da keinen Dokbook-terminus für?.

Reply:
Wenn im Guide für die Menüeinträge auch die neuen Entities eingearbeitet sind, dann erledigt sich dieses Problem von selber.